components/TrendingCollectionTable.tsx

```diff
@@ -90,7 +90,7 @@ const TrendingCollectionTable: FC<Props> = ({ fallback }) => {
               <tr
                 key={`${contract}-${index}`}
                 ref={index === arr.length - 5 && shouldInfiniteLoad ? ref : null}
-                className={`${index === arr.length - 1 || (!expanded && index == 9) ? '' : 'border-b'} group h-[88px] border-neutral-300 dark:border-neutral-600 dark:text-white`} 
+                className={`${index === arr.length - 1 || (FOOTER_ENABLED && !expanded && index == 9) ? '' : 'border-b'} group h-[88px] border-neutral-300 dark:border-neutral-600 dark:text-white`} 
```

Comment:
Need to cast this to a boolean before checking, like we discussed a little while ago.